### ISSUE TITLE
Unable to launch tetrad-gui-6.9.0-launch.jar in Windows 10

### ISSUE THREAD

**Issue by cagataykiyici** [cmu-phil/tetrad#1357]
Hi, I'm a new user and trying to use Tetrad first time. Unfortunately, I can not launch the Java GUI app.

I'm using a Windows10 (21H2 latest version all updates installed) laptop and latest Java version 8 update 311 (build 1.8.0_311-b11). I'm the administrator, no access problems. 

When I launch it this screen is displayed and after long time waiting nothing happens. 
![Screenshot 2021-12-15 082826](https://user-images.githubusercontent.com/17346159/146129540-a965f654-1d0c-440e-a5a2-a467154ce169.png)

When I launch from cmd with "run as administrator" option and run java -jar C:\tetrad-gui-6.8.0-launch.jar same screen and following errors on cmd screen. 
![Screenshot 2021-12-15 083054](https://user-images.githubusercontent.com/17346159/146129578-1ae566aa-4f89-4f10-941a-9e34d313bc75.png)

Full text of cmd screen errors are in this txt file: 
[console.txt](https://github.com/cmu-phil/tetrad/files/7716697/console.txt)

Also, tried adding skip latest option, and I also tried to run 6.8.1 and 6.8.0 but still failed to launch the software. (I always reboot computer before running again)

I'm quite new to causal inference area and excited to use Tetrad for a graduate project. 

Thanks in advance for your help. 

Best

**Comment by jdramsey** [comment]
I'm wondering if the server that checks connections is down maybe... @espinoj ? @kvb2univpitt ?

I don't have the ability to check that myself. If there's no one around to fix it, I can put a version of Tetrad up somewhere for separate download.

**Comment by cg09** [comment]
Connections to *what*?

**Comment by jdramsey** [comment]
It seems @cagataykiyici is having trouble launching the Tetrad interface because the check in the code that checks against the server to count users of the interface isn't working properly. I can disable that, but I don't have the ability to post an update--I need some help for that.

Daniel Yuan was also having trouble, so it might not be specified to @cagataykiyici .

**Comment by jdramsey** [comment]
BY THE WAY-----I have a version of Tetrad 6.9.0 built on my machine with this hotfix---if @cagataykiyici or any others out there need it before we can get it posted, please contact me directly. Thanks! Joe Ramsey jdramsey@andrew.cmu.edu.

**Comment by cg09** [comment]
I have forgotten how to find the tetrad versions (not the source codes) on
github. Help!

Clark

**Comment by jdramsey** [comment]
Honestly, it won't do you much good until we can fix the bug. Shall I send you a copy of the fix?

**Comment by cg09** [comment]
If it includes the interface, sure, love to have it.

Clark

**Comment by cagataykiyici** [comment]
> BY THE WAY-----I have a version of Tetrad 6.9.0 built on my machine with this hotfix---if @cagataykiyici or any others out there need it before we can get it posted, please contact me directly. Thanks! Joe Ramsey [jdramsey@andrew.cmu.edu](mailto:jdramsey@andrew.cmu.edu).

Thank you very much, I have just sent an email :)

**Comment by jdramsey** [comment]
@kvb2univpitt I think this is resolved, no? Has this been published? If so, I'll tell Clark and the others.

**Comment by jdramsey** [comment]
Fixed, you just need Java JDK 8.
